### PR TITLE
Setup GitPod for easy anywhere coding

### PR DESCRIPTION
**Pull request by TheFehr**

### CODE REVIEW

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM gitpod/workspace-postgres
+
+# Install Ruby
+ENV RUBY_VERSION=2.7.1
+RUN rm /home/gitpod/.rvmrc && touch /home/gitpod/.rvmrc && echo "rvm_gems_path=/home/gitpod/.rvm" > /home/gitpod/.rvmrc
+RUN bash -lc "rvm install ruby-$RUBY_VERSION && rvm use ruby-$RUBY_VERSION --default"
+
+RUN bash -lc "mkdir -p /home/gitpod/.sdkman/candidates/java/current/jre/lib/amd64/server"
+RUN bash -lc "sudo ln -s /home/gitpod/.sdkman/candidates/java/current/lib/server/libjvm.so /home/gitpod/.sdkman/candidates/java/current/jre/lib/amd64/server/libjvm.so"
```
Comment (lukasbischof) [inline .gitpod.Dockerfile:9]:
```suggestion
RUN bash -lc "sudo ln -s /home/gitpod/.sdkman/candidates/java/current/lib/server/libjvm.so /home/gitpod/.sdkman/candidates/java/current/jre/lib/amd64/server/libjvm.so"
```

Comment (lukasbischof) [reply .gitpod.Dockerfile:9]: No newline

File: config/environments/development.rb [hunk 2]
```diff
@@ -59,4 +59,6 @@
     Bullet.bullet_logger = true
     Bullet.add_footer = true
   end
+
+  config.hosts = nil
```
Comment (lukasbischof) [inline config/environments/development.rb:63]: Why not configuring `ENV['APP_HOST']`?

Comment (TheFehr) [reply config/environments/development.rb:63]: I mean sure for test and prod. But why would we need a host limitation in dev?

Comment (lukasbischof) [reply config/environments/development.rb:63]: So that the evil hackerman can't spoil your development system

File: README.md [hunk 3]
```diff
@@ -1,24 +1,3 @@
...
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+# MyZivi
```
Comment (lukasbischof) [inline README.md:3]: What a sophisticated README 🙈

**Pull request merged**